float32[] trans
float32[] rot
string incrimental
string change_height
string keep_orient
---
string success